src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -48,6 +48,13 @@ class DockerCreateContainer extends DockerExistingImage {
     @Optional
     final Property<String> user = project.objects.property(String)
 
+    /**
+     * A list of additional groups that the container process will run as.
+     */
```

Comment:
Can you add the `@since` annotation with the next version so users know when this field was introduced?

Reply:
I thought about that, but what would be next version number?

Reply:
Correct, I already assigned the milestone to the issue.